uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -566,6 +567,39 @@ define([
               });
             });
           });
+
+          describe('when user edits the request', function () {
+            beforeEach(function () {
+              var status = optionGroupMock.specificValue(
+                'hrleaveandabsences_leave_request_status', 'value', '3');
+              var leaveRequest = LeaveRequestInstance.init(
+                leaveRequestData.findBy('status_id', status));
+
+              selectedAbsenceType.calculation_unit_name = 'hours';
+              leaveRequest.contact_id = '' + CRM.vars.leaveAndAbsences.contactId;
```

Comment:
why ` '' + ` ?

Reply:
Converting to a string instead of .toString()